softwareComponents/rofiHalSim/wait_worker.hpp

```diff
@@ -38,13 +38,23 @@ class WaitCallbacks
     {
         assert( callback );
         std::lock_guard< std::mutex > lock( _mutex );
-        _callbacks.try_emplace( waitId, std::move( callback ) );
+        [[maybe_unused]] auto success = _callbacks.try_emplace( waitId, std::move( callback ) );
```

Comment:
K čemu je zde `[[maybe_unused]]`?

Reply:
Původně tam byl assert. Asi to pak oddělám, teď to mergnu.